services/server/src/server/server.ts

```diff
@@ -315,6 +384,9 @@ if (require.main === module) {
       server.app.listen(server.port, () => {
         logger.info("Server listening", { port: server.port });
       });
+    })
+    .then(async () => {
```

Comment:
Small thing but this should be after `const server = new Server()`, and not after the swagger calls and especially after the "Server listening" call. The listening call shoudl be the very last thing.